cmd/ci-reporter/cmd/root.go

```diff
@@ -89,7 +89,10 @@ func RunReport(cfg ReporterConfig) error {
 	if err != nil {
 		return err
 	}
-	PrintTestgridReportData(testgridReportData)
+	err = PrintTestgridReportData(cfg, &testgridReportData)
+	if err != nil {
+		return err
+	}
```

Comment:
nit
```suggestion
	if err = PrintTestgridReportData(cfg, &testgridReportData); err != nil {
		return err
	}
```

Reply:
I am getting this linting warning:
sloppyReassign: re-assignment to `err` can be replaced with `err := PrintTestgridReportData(cfg, &testgridReportData)`

Reply:
oops yes my bad there